base/src/main/java/com/smartdevicelink/proxy/rpc/enums/AppCapabilityType.java

```diff
@@ -0,0 +1,23 @@
+package com.smartdevicelink.proxy.rpc.enums;
```

Comment:
@kostyaBoss I generated this file using the generator tool located at: https://github.com/smartdevicelink/sdl_java_suite/tree/master/generator

I see differences between the generated file and this file.  Can you please generate this file using the tool?

Reply:
@santanamk, unfortunately, I was unable to generate this file. The exception I've got from the generator is

12-13 17:35 - Generator - CRITICAL - Invalid XML file content: /Users/KBoskin/Desktop/luxoft/rpc_spec/MOBILE_API.xml, Unknown type 'VideoStreamingCapability'

The base I've used for 0296 spec API was https://github.com/LuxoftSDL/rpc_spec.git